htdocs/admin/delais.php

```diff
@@ -354,21 +354,21 @@
 
 		print '</div>';
 
-		print '<div id="percentage" '.(empty($conf->global->MAIN_USE_METEO_WITH_PERCENTAGE) ? 'style="display:none;"' : '').'>';
+		print '<div id="percentage" '.getDolGlobalString('MAIN_USE_METEO_WITH_PERCENTAGE').'>';
```

Comment:
The style="display:none;" shown if MAIN_USE_METEO_WITH_PERCENTAGE is empty seems lost.